tests/worktree_bare_repo_spec.lua

```diff
@@ -0,0 +1,94 @@
+local git_worktree = require('git-worktree')
+local Path = require('plenary.path')
+
+local harness = require('tests.git_harness')
+local in_bare_repo_from_origin_no_worktrees = harness.in_bare_repo_from_origin_no_worktrees
+local in_bare_repo_from_origin_1_worktree = harness.in_bare_repo_from_origin_1_worktree
+
+local completed_create = false
+local completed_switch = false
+local completed_delete = false
+
+local reset_variables = function()
+    completed_create = false
+    completed_switch = false
+    completed_delete = false
+end
+
+describe('git-worktree bare repo', function()
+
+    it('can create a worktree from a bare repo and switch to it', in_bare_repo_from_origin_no_worktrees(function()
+
+        reset_variables()
```

Comment:
I don't think there is a beforeEach, but this would be a nice candidate :)

Reply:
is there, I can check . still learning busted